### DISCUSSION TITLE
Upload (small) files

### DISCUSSION THREAD

**Discussion by Solear** [seanmorley15/AdventureLog#371] [Ideas]
Hello! I think the app is great. I installed it using the tteck (RIP) script. It works wonderfully so far, except that I'm having trouble with Caddy, but it works via the IP address.  
My suggestion: The ability to upload small files (PDFs, eml, doc, txt). Then you could manage the entire travel planning with adventurelog, for example, by adding restaurant bookings or printing the train route.  
I know we have to be careful since some people use it publicly, and that can be risky.  
But I would love this feature!

**Comment by seanmorley15** [comment]
All set in the recent update :)
